README.md

```diff
@@ -871,7 +871,7 @@ When documenting an unstable feature, those unstable packages must be explicitly
 <PackageReference Include="NServiceBus.RabbitMQ" Version="4.3.1-alpha.XYZ" />
 ```
 
-Wildcard patterns, like `Version="4.3.1-*` are not allowed for pre-releases since they tend to break the snippet and sample builds.
+Wildcard patterns, like `4.3.1-*`, or `8.0.0-alphe/beta/rc.*` are not allowed for pre-releases since they tend to break the snippet and sample builds.
```

Comment:
@danielmarbach It was not clear if that was allowed so I clarified by explicitly included prerelease names.